include/pmacc/particles/memory/frames/Frame.hpp

```diff
@@ -125,15 +102,15 @@ namespace pmacc
          * @return result of operator[] of MapTupel
          */
         template<typename T_Key>
-        HDINLINE typename boost::result_of<ThisType(T_Key)>::type getIdentifier(const T_Key)
+        HDINLINE auto& getIdentifier(const T_Key)
         {
             using Key = typename GetKeyFromAlias<ValueTypeSeq, T_Key>::type;
```

Comment:
please add `errorHandlerPolicies::ThrowValueNotFound` to `GetKeyFromAlias`, The throw policy was handled in the removed `result struct`. The last policy is defining whats happens if the key is not available in the Frame.

Reply:
done!